### ISSUE TITLE
[Quest] Betrayal

### ISSUE THREAD

**Issue by jarvik76** [WoWManiaUK/Redemption#2950]
http://www.wow-mania.com/armory/?quest=12713

For this quest you'll need to kill Drakuru(http://www.wow-mania.com/armory/?npc=28503) making use of the Scepter of Domination(http://www.wow-mania.com/armory/?item=39664) to place a Blightblood Troll(http://www.wow-mania.com/armory/?npc=28931) under your control. After Drakuru has been defeated Drakuru's Skull(http://www.wow-mania.com/armory/?item=40425) will spawn.

Whilst there's been many tickets about having to be exactly in the right spot, I experienced a new issue - about 3 seconds after Drakuru's death, his ability Blight Fog starts happening again.  Because there is a channel time to pick up the skull, you're hit with damage before you can pick up the skull.  Even if you do somehow manage this in time, the damage continues even after this - effectively it is a guaranteed death.  If you release spirit, the Blight Fog continues at the graveyard.

This issue was witnessed by GM Psygoa, they stated that even if they removed the aura, it came back within a few seconds.  Am raising ticket on their suggestion.

Client 2.5.0 using the launcher, cleaned cache.

Secondarily, if you then teleport back to the Reliquary after you were killed by the Fog, after you killed Drakuru, but BEFORE he respawns, you can speak to Drakuru manually, you are teleported up to the platform but then immediately teleported back down.  If you fly manually to try to pick up the skull, again you are teleported back down.  In other words, there is no way to get back to the platform successfully if you die from the Fog after Drakuru has been killed.

**Comment by gamer100** [comment]
I Have seen it happen  i  got also the blight Fog when i removed it , it was back in 2 seconds

**Comment by wow-mania** [comment]
Although there are some quirks on this quests it is totally doable as is.
We will look at it when we have time. Messing with it too much may totally break it so it is not a priority.

**Comment by Dranah** [comment]
I have just tested this quest and reproduced issue.
- Once the event is complete, Drakuru appears to respawn immediately but he's phased (probably that's intended since you no longer need to be in his phase, you're already done)
- The problem is that this Drakuru, despite being in another phase, is constantly attempting to engage the player every ~2sec. He emotes "What is this treachery?..blabla" every single time, but obviously as he's in a different phase, his script cannot carry on, he keeps dropping combat right away. And every time he tries to engage the player, it applies the Blight Fog aura that he normally casts on the pull. spell ID 54105
This is how he looks from GM-mode, regular players can't see him as he's not in your phase: https://imgur.com/UMF57ye
- Because he applies a new aura every 2s, GM removing aura manually from the player will not help, nor the player is able to move out of the fog as it spawns a new one on him every time the aura is reapplied.
This is a guaranteed death.
- While the skull can be picked up if you're quick, if you somehow miss your chance to take it before the blight kills you off, you'll probably have to redo the whole quest. After rez, I did not get an option from Drakuru to be teleported to the upper platform again.
- I was able to fly back up using a fly mount (which the average lvl 75 player lvling in zd does NOT have), however upon landing, any lowbies would be instantly killed by the invisible Drakuru.

TL;DR: the real problem here seems to be that the event tries to re-engage from a different phase for no reason
The quest may be doable, but if you do it wrong, you will have to start all over with no guarantee that you won't fail it again. And killing a 500k elite is time consuming, this means nobody will actually keep trying.

**Comment by Dranah** [comment]
Sure it is no high priority, but it's not like we have ten thousand reports open atm.

**Comment by wow-mania** [comment]
I have just done it and it worked.
I have done all the way to the end.

**Comment by wow-mania** [comment]
drakuru is not even there permanently anymore

**Comment by Dranah** [comment]
I tested it 5mins ago.... lemme log

**Comment by wow-mania** [comment]
Fixed as per Drannah comments

**Comment by Dranah** [comment]
UPDATE:
- Drakuru no longer respawns and begins attacking players immediately after the event has been completed. (fixed)
- Event has now a 4-5mins of cooldown, which means during this time other players who teleport to the platform won't be able to begin the event (drakuru wont be spawned yet) (needs tweaking, imho this is needlessly long)
- Blight fog is still not fully functional, but will no longer murder players with a permanent aura (fixed, needs tweaking)
- Lich King cuts off his speech and despawns the moment the player leaves combat after he murders Drakuru (tweaking issue, cosmetic)
- Quest item is lootable without fogs hitting you and the object to teleport to the ground is now spawned near the back of platform. (Drakuru's Last Wish)

Summary:
Overall the quest can now be completed without much problem, but still has many tweaking glitches.
I'd suggest the timer for event respawn to be shortened so players don't have to wait that long.

**Comment by wow-mania** [comment]
Both issues fixed.

**Comment by Dranah** [comment]
UPDATE:
Last fix broke it again and now we're back to where we started. Drakuru respawns immediately and begins trying to attack you from the other phase with a fog.
Please revert this and just try to reduce the timer.

**Comment by jarvik76** [comment]
Thanks for fixing this guys - makes me feel it was worth the time reporting it.  Thanks!
